src/index.js

```diff
@@ -106,15 +106,17 @@ function jsxToString (component, options) {
         value = opts.keyValueOverride[key](component.props[key]);
       } else if (opts.keyValueOverride[key]) {
         value = opts.keyValueOverride[key]
+      } else if (opts.shortBooleanSyntax && typeof component.props[key] === 'boolean') {
```

Comment:
I believe we have to account for the differences between `undefined` and `false`. If a boolean value is defined but set to `false` I don't think it should be ignored from the output, meaning the shortBooleanSyntax is only applied for "truthy" values. Thoughts?

The reason I want to account for `undefined` vs `false` is when a value is undefined it can use defaultProps. For example, let's say we have a boolean `responsive` prop to a component (Box) where the default value is `true`, so you can use the following syntax:

```
<Box responsive>
</Box>
<Box responsive={true}>
</Box>
<Box />
```

In case you want to disable the responsive capabilities of the Box you do:

```
<Box responsive={false} />
```

If we merge this PR, I believe it will yield `<Box />` instead of `<Box responsive={false} />`.=

Reply:
I maybe followed the #29 example too rigorously.

Indeed by removing the falsy props, we actually potentially hide a useful prop.
Plus removing it should never be a real desire of jsx-to-string users.

To be clear, if
```js
<Box responsive />            //=> <Box responsive />
<Box responsive={true} />     //=> <Box responsive />
<Box responsive={false} />    //=> <Box responsive={false} />
```
should be the actual behavior of this feature, I'll correct the PR.

Reply:
I have a commit ready for it @alansouzati if you validate the choice

Reply:
sure it sounds good to me. as soon as you send the updated commit I will review it again. thanks!!